cdk/lib/__snapshots__/prism-ec2-app.test.ts.snap

```diff
@@ -906,6 +906,10 @@ dpkg -i /prism/prism.deb",
             "Key": "deregistration_delay.timeout_seconds",
             "Value": "30",
           },
+          Object {
+            "Key": "stickiness.enabled",
+            "Value": "false",
```

Comment:
`false` was the implicit [default](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-targetgroup-targetgroupattribute.html) previously - this change should be a no-op.